packages/strapi-bookshelf/lib/index.js

```diff
@@ -361,7 +361,7 @@ module.exports = function(strapi) {
                             type = definition.client === 'pg' ? 'double precision' : 'double';
                             break;
                           case 'decimal':
-                            type = 'decimal';
+                            type = 'decimal(10,2)';
```

Comment:
I think we should look at making this configurable in the adminUI and the config files if possible. 

Using this I believe the accuracy is 10 total digits with 2 being after the decimal point so something like:
`99999999.99`

Reply:
@derrickmehaffy yes it's a good idea ! But in an other PR. Can you create an issue (feature request) with detail of this options. Thank you !

The topic of this one is just to make it work ;)